selfdrive/car/chrysler/carcontroller.py

```diff
@@ -39,22 +40,19 @@ def __init__(self, dbc_name, car_fingerprint, enable_camera):
   def update(self, sendcan, enabled, CS, frame, actuators,
              pcm_cancel_cmd, hud_alert, audible_alert):
 
-    # this seems needed to avoid steerign faults and to force the sync with the EPS counter
+    # this seems needed to avoid steering faults and to force the sync with the EPS counter
     if self.prev_frame == frame:
       return
 
     # *** compute control surfaces ***
     # steer torque
     apply_steer = actuators.steer * SteerLimitParams.STEER_MAX
-    apply_steer = apply_toyota_steer_torque_limits(apply_steer, self.apply_steer_last,
-                                                   CS.steer_torque_motor, SteerLimitParams)
-
     moving_fast = CS.v_ego > CS.CP.minSteerSpeed  # for status message
     lkas_active = moving_fast and enabled
-
     if not lkas_active:
       apply_steer = 0
-
+    apply_steer = apply_toyota_steer_torque_limits(apply_steer, self.apply_steer_last,
+                                                   CS.steer_torque_motor, SteerLimitParams)
```

Comment:
Mmm... I think panda does not let any non-zero going through immediately after disengagement